From b7bbf556424016d1ac022c4ff9536a9c357daff7 Mon Sep 17 00:00:00 2001
From: Scott Ikenaga <sikenaga@ueidaq.com>
Date: Mon, 24 Jun 2024 13:17:07 -0500
Subject: [PATCH] Remove rule configure_crypto_policy

- Rule checks for a Redhat specific package which is not instlaled
---
 products/openembedded/profiles/uei.profile | 2 --
 1 file changed, 2 deletions(-)

diff --git a/products/openembedded/profiles/uei.profile b/products/openembedded/profiles/uei.profile
index 7bbe41c57f..a55f4b2f75 100644
--- a/products/openembedded/profiles/uei.profile
+++ b/products/openembedded/profiles/uei.profile
@@ -126,8 +126,6 @@ selections:
     - require_singleuser_auth
     - require_emergency_target_auth
     - disable_users_coredumps
-    - configure_crypto_policy
-    - var_system_crypto_policy=default_policy
     - dir_perms_world_writable_sticky_bits
     - file_permissions_etc_passwd
     - file_owner_etc_shadow
